src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/methodname/InputMethodNameRecordInInterfaceBody.java

```diff
@@ -0,0 +1,29 @@
+/*
+MethodName
+format = (default)^[a-z][a-zA-Z0-9]*$
+allowClassName = (default)false
+applyToPublic = false
```

Comment:
it should be set to default value = true, otherwise record member wont be checked (they are public fields)

Reply:
Done